documentation/development/CodingStandards.md

```diff
@@ -5,49 +5,59 @@ Coding Style and Standards for Falaise {#codingstandards}
 
 Introduction {#coding_intro}
 ============
-The key guidelines for Falaise code are **consistency**, **readability**
-and **C++ standard compliance**. The aim is not to fight wars
-over minutiae, but provide an easy to use and common guide for how
-to layout, name and implement code.
+This document is a work in progress, and is expected to evolve
+through both the lifetime of the Falaise software as well as
+the now regular updates to the C++ language it is written in.
 
-This guide is still a work in progress. If in doubt, use what's in the
-file you're working on already. If you're starting from scratch,
-read the sections below!
+With language and design being an ever changing landscape, the
+key guidelines when writing code for Falaise are
 
-Basic Style Guide
------------------
-Falaise code should follow the [Google C++ Style Guide](http://google-styleguide.googlecode.com/svn/trunk/cppguide.xml). The [C++ Coding Standards](http://www.gotw.ca/publications/c++cs.htm) textbook is also used to
-provide guidance on writing production quality code.
+- **consistency** (mutual understanding)
+- **readability** (clarity of intent)
+- **C++ standard compliance** (reliability)
 
 
 Consistency
 -----------
-Whatever coding style you use, it **must be consistent** throughout the
-code. Layout code blocks using the same style throughout your code.
-Don't swap styles between files, or worse, halfway through a file.
-If a style is already present in a file you are working on, use that
-style.
+Falaise code should follow the [Google C++ Style Guide](http://google-styleguide.googlecode.com/svn/trunk/cppguide.xml) for spacing, layout and organization of classes.
+To help apply this formatting, use of the [clang-format tool](https://clang.llvm.org/docs/ClangFormat.html) is recommended.
+A `.clang-format` file for Falaise is supplied at the root of the source tree.
+Integration of `clang-format` with your favourite editor or IDE is easy, with guides
+available in the [clang-format documentation](https://clang.llvm.org/docs/ClangFormat.html).
+Falaise's build system can also run `clang-format` for you. Ensure that `clang-format`
+is installed and available in your path, then
+
+```
+$ cmake <sourcedir>
+$ make clang_format
```

Comment:
`clang_format` is just another target, so `ninja clang_format` should work just as well.